addons/website/models/theme_models.py

```diff
@@ -209,6 +209,15 @@ def _reset_default_config(self):
         # Reinitialize footer scrolltop template
         self.disable_view('website.option_footer_scrolltop')
 
+    @api.model
+    def _toggle_asset(self, name, active):
+        website = self.env['website'].get_current_website()
+        assets = self.env['ir.asset'].search([('name', '=', name), ('website_id', '=', website.id)])
```

Comment:
search without active_test incontext doesn't return inactive record

=> assets.write({}) should be the same